server/utils/factories.js

```diff
@@ -12,12 +12,17 @@ const uuid = {
 
 const fixtures = {
   user(user = {}) {
-    return {
+    const obj = {
       name: user.name || Faker.internet.userName(),
       email: user.email || Faker.internet.email(),
-      groups: user.groups,
       isSysAdmin: user.isSysAdmin || false
     };
+
+    if (user.groups) {
+      obj.groups = user.groups;
+    }
+
+    return obj;
```

Comment:
We can't pass groups in the object to some of the functions, so this makes sure that the fixtures don't give us an object that has groups in it.